6-anthos-install/README.md

```diff
@@ -0,0 +1,353 @@
+kubectl --context ${CTX_INGRESS} -n istio-system apply -f ${HOME}/mcs.yaml
+```
+
+## Install and Configure ACM
+## Create a Private Key
+```console
+kubectl create ns config-management-system --context ${CTX_1} && kubectl create secret generic git-creds --namespace=config-management-system --context ${CTX_1} --from-file=ssh="id_rsa.nomos"
+
+kubectl create ns config-management-system --context ${CTX_2} && kubectl create secret generic git-creds --namespace=config-management-system --context ${CTX_2} --from-file=ssh="id_rsa.nomos"
+```
+
+### Download and install ACM
+
+```console
+gsutil cp gs://config-management-release/released/1.7.0/config-management-operator.yaml config-management-operator.yaml
+
+kubectl apply --context=${CTX_1} -f config-management-operator.yaml
+kubectl apply --context=${CTX_2} -f config-management-operator.yaml
+```
+
+### Configure ACM 
+```console
+kubectl apply --context=${CTX_1} -f ${HOME}/terraform-example-foundation-app/acm-configs/config-management-gke-east.yaml
+
+kubectl apply --context=${CTX_2} -f ${HOME}/terraform-example-foundation-app/acm-config/config-management-gke-west.yaml
+```
+
+### Populate the CSR repos 
+#### root config repo
+1. Copy the content of `acm-repos/root-config-repo` to `${HOME}/root-config-repo`
+```console
+cd ${HOME}
+cp ${HOME}/terraform-example-foundation-app/acm-repos/root-config-repo ${HOME}/root-config-repo
+```
+
+2. Move to the new folder
+```console
+cd ${HOME}/root-config-repo 
+```
+3. push the content to the root-config-repo
+
+```console
+gcloud source repos clone config-root-repo
+git add .
+git commit -m “adding config repo”
+git push origin master
+```
+#### accounts namespace
+1. Copy the content of `acm-repos/accounts` to `${HOME}/accounts`
+```console
+cd ${HOME}
+cp ${HOME}/terraform-example-foundation-app/acm-repos/accounts ${HOME}/accounts
+```
+
+2. Move to the new folder
+```console
+cd ${HOME}/accounts
+```
+3. push the content to the accounts repo
+
+```console
+gcloud source repos clone accounts
+git add .
+git commit -m “adding accounts repo”
+git push origin master
+```
+#### frontend namespace
+1. Copy the content of `acm-repos/frontend` to `${HOME}/frontend`
+```console
+cd ${HOME}
+cp ${HOME}/terraform-example-foundation-app/acm-repos/frontend ${HOME}/frontend
+```
+
+2. Move to the new folder
+```console
+cd ${HOME}/frontend
+```
+3. push the content to the frontend repo
+
+```console
+gcloud source repos clone frontend
+git add .
+git commit -m “adding frontend repo”
+git push origin master
+```
+
+#### transactions namespace
+1. Copy the content of `acm-repos/transactions` to `${HOME}/transactions`
+```console
+cd ${HOME}
+cp ${HOME}/terraform-example-foundation-app/acm-repos/transactions ${HOME}/transactions
+```
+
+2. Move to the new folder
+```console
+cd ${HOME}/transactions
+```
+3. push the content to the transactions repo
+
+```console
+gcloud source repos clone transactions
```

Comment:
In the latest commit is see some git submodules added for these repos. Should we just have instructions to clone from upstream?